src/common/idle_inhibitor.cpp

```diff
@@ -0,0 +1,231 @@
+// Copyright © 2021 Steven A. Wilson
+
+// Permission to use, copy, modify, and/or distribute this software
+// for any purpose with or without fee is hereby granted.
+
+// THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL
+// WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED
+// WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE
+// AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR
+// CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS
+// OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT,
+// NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
+// CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+#include "idle_inhibitor.h"
+
+#include <cstdlib>
+
+#include "common/logging/log.h"
+
+#if defined(_WIN32)
+#include <windows.h>
+#elif defined(__unix__)
+#include <chrono>
+#include <dlfcn.h>
+#include <unistd.h>
+struct DBusConnection;
+struct DBusError;
+struct DBusMessage;
+typedef uint32_t dbus_bool_t;
+typedef uint32_t dbus_uint32_t;
+#endif
+
+namespace {
+#if defined(__unix__)
+const char* g_app = "yuzu";
+const char* g_reason = "Emulation is running";
+#endif
+
+#if defined(__unix__)
+const int DBUS_TYPE_INVALID = '\0';
+const int DBUS_TYPE_STRING = 's';
+const int DBUS_TYPE_UINT32 = 'u';
+const int DBUS_TIMEOUT_USE_DEFAULT = -1;
+enum DBusBusType { DBUS_BUS_SESSION, DBUS_BUS_SYSTEM, DBUS_BUS_STARTER };
```

Comment:
```suggestion
enum DBusBusType { DBUS_BUS_SESSION, DBUS_BUS_SYSTEM, DBUS_BUS_STARTER, };
```

So that clang format doesn't single line the enum.